lib/ui/screens/new_contact_screen.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:flutter/material.dart';
+
+class NewContact extends StatefulWidget {
+    const NewContact({Key? key }) : super(key: key);
+
+    @override
+    _NewContactState createState() => _NewContactState();
+}
+
+class _NewContactState extends State<NewContact> {
+    late String firstName; 
+    late String lastName; 
+    late String phoneNumber; 
+    String gender = ""; 
+    late String email; 
+    late String address; 
+
+    List<String> genderList = ["Male", "Female", "Others"]; 
```

Comment:
Should create enum type like this:
`enum Gender { Male, Female, Others }

extension GenderName on Gender {
  String get name {
    switch (this) {
      case Gender.Male:
        return 'Male';
      case Gender.Female:
        return 'Female';
      case Gender.Others:
        return 'Others';
    }
  }
}`